### DISCUSSION TITLE
One Channel [GR]

### DISCUSSION THREAD

**Discussion by GreekGamerTheGreat** [iptv-org/community#372] [Show and tell]
### Channel ID

OneChannel.gr

### Stream URL

https://streaming.onetv.gr/show/onetv_src/index.m3u8

### Notes

Will Close If Added

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by AntiPontifex** [comment]
Already on the list: please close.

**Comment by GreekGamerTheGreat** [comment]
> Already on the list: please close.
The One Channel Im Requesting To Add Is From A Completely Diffrent Country From The Channels I Saw Existed Are You Sure There Is One Channel From Greece Already Added?

**Comment by freearhey** [comment]
https://streaming.onetv.gr/show/onetv_src/index.m3u8 (offline)
<img width="382" alt="image" src="https://user-images.githubusercontent.com/7253922/232254179-f278371c-684e-4da1-aa3e-3c5aa719d49a.png">
